docs/intro/install.rst

```diff
@@ -72,7 +72,7 @@ please refer to their respective installation instructions:
 * `cryptography installation`_
 
 .. _lxml installation: https://lxml.de/installation.html
-.. _cryptography installation: https://cryptography.io/en/latest/installation/
+.. _cryptography installation: https://cryptography.io/en/latest/installation.html
```

Comment:
Can we please switch to InterSphinx to handle this?

If we add an entry for `cryptography` to [`intersphinx_mapping`](https://github.com/scrapy/scrapy/blob/master/docs/conf.py#L283) we should be able to link with something like ```:doc:`cryptography:installation` ```.

Reply:
Hi @Gallaecio, This is my first time working with intersphinx so please let me know I made a mistake.